src/PackageServiceProvider.php

```diff
@@ -87,8 +87,8 @@ public function registerEvents()
                     $v->addHeaderItem($asset);
 
                     $v->requireAsset('javascript', 'free_cookies_disclosure/disclosure_hide');
-                    $v->requireAsset('javascript', 'free_cookies_disclosure/disclosure_ajax_form');
                 }
+                $v->requireAsset('javascript', 'free_cookies_disclosure/disclosure_ajax_form');
```

Comment:
Thanks for the PR! Can we move this to the place where we have the other `requireAsset` calls inside this if-clause? This might be clearer to other developers coming back to this later on? I mean below the line that says `$v->requireAsset('javascript', 'jquery');`.